### ISSUE TITLE
ngx-smart-modal v7.2.0 not working

### ISSUE THREAD

**Issue by vitokodev** [biig-io/ngx-smart-modal#233]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
The web browser console throws:
```
Uncaught Error: Can't resolve all parameters for NgxSmartModalService: (?, ?, ?, ?, ?).
    at syntaxError (compiler.js:2175)
    at CompileMetadataResolver._getDependenciesMetadata (compiler.js:20399)
    at CompileMetadataResolver._getTypeMetadata (compiler.js:20294)
    at CompileMetadataResolver._getInjectableTypeMetadata (compiler.js:20512)
    at CompileMetadataResolver.getProviderMetadata (compiler.js:20521)
    at compiler.js:20459
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver._getProvidersMetadata (compiler.js:20419)
    at compiler.js:20050
    at Array.forEach (<anonymous>)
```

**Expected behavior**
Load the default new Angular project including ```NgxSmartModalModule``` without errors.

**Reproduction of the problem**

I created a new Angular project without routing option, then ran ```ng serve -o``` so I can make sure the project is running which is what happened.

Finally, I edited app.module.ts:
```
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { NgxSmartModalModule } from 'ngx-smart-modal';

import { AppComponent } from './app.component';

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule,
    NgxSmartModalModule.forRoot()
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```
Then the app broke.

**What is the motivation / use case for changing the behavior?**

I tried to use it in my project and I did not find any solution so I decided to try this versión with a new clean project.

**Please tell us about your environment:**

Windows 10. NPM 6.9.0. NodeJS 10.16.0

* **Smart Modal version:** 0.8.x

7.2.0

* **Angular version:** 2.0.x

7.3.9 and 8.3.6

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

all

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all

**Comment by 49e94b8f256530dc0d41f740dfe8a4c1** [comment]
I also had to downgrade to 7.1.1

**Comment by hamony** [comment]
The same issue even I upgrade Angular to version 8.2.8

**Comment by andres01gc** [comment]
the same problem :(

**Comment by maximelafarie** [comment]
I'm so sorry for the inconvenience, however we've tested it so many times before releasing.
I'm working now on an emergency patch release.
I'll let you all know when it is available.
Regards 🖖

**Comment by maximelafarie** [comment]
Hi all!
The `v7.2.1` has just been released and fixes the issue (#234)! 🎉 I would like to thank you all for reporting us this error and please forgive us for this bad release!
Thank you very much for your support! 🙏☺️❤️

![image](https://user-images.githubusercontent.com/5319267/65878296-0e857800-e38e-11e9-8e2c-987aa34e5edc.png)


Have a nice day,
Best regards! 🖖
